gaps/gap-3_mid_agreement_payments/gap-3_mid_agreement_payments.md

```diff
@@ -0,0 +1,91 @@
+---
+gap: 3
+title: Mid Agreement payments
+description: Specification of Agreement standard for negotiating and describing continues payments during task execution.  
+author: nieznany.sprawiciel (@nieznanysprawiciel), stranger80 (@stranger80), shadeofblue (@shadeofblue)
+status: Draft
+type: Feature
```

Comment:
@stranger80 Why it is feature, not standard?

Reply:
To me it is a feature as it requires changes in components which are to implement this logic. A 'standard' amendment would only improve or publish standards but not provide 'features' in software.